regulations/static/regulations/css/less/module/comment-read.less

```diff
@@ -117,7 +112,8 @@ Preamble Read Mode
   }
 
   .cfr-instructions {
-    .font-regular;
+    font-family: "Merriweather", Georgia, serif;
+    font-weight: 400;
```

Comment:
This was a temporary thing where I copied a lone rule directly from n&c as to not break the design until the right font mixins are established in -site. I plan to swing back around to this and other spots where font families are declared directly in the next phase.